hail_scripts/reference_data/clinvar.py

```diff
@@ -117,19 +112,19 @@ def download_and_import_latest_clinvar_vcf(
 
     if genome_version not in ['37', '38']:
         raise ValueError('Invalid genome_version: ' + str(genome_version))
-    clinvar_url = CLINVAR_FTP_PATH.format(genome_version=genome_version)
-    urllib.request.urlretrieve(clinvar_url, tmp_file.name)  # noqa: S310
-    clinvar_release_date = _parse_clinvar_release_date(tmp_file.name)
     mt_contig_recoding = {'MT': 'chrM'} if genome_version == '38' else None
-    mt = import_vcf(
-        tmp_file.name,
-        genome_version,
-        drop_samples=True,
-        min_partitions=2000,
-        skip_invalid_loci=True,
-        more_contig_recoding=mt_contig_recoding,
-    )
-    return mt.annotate_globals(version=clinvar_release_date)
+    with tempfile.NamedTemporaryFile(suffix='.vcf.gz', delete=False) as tmp_file:
```

Comment:
Thinking about `tempfile` cleanup was painful here so we're back to leaving a file behind.  Maybe the cleaner answer here is to checkpoint to `gs://seqr-scratch-temp`?  

I played around with `hl.import_vcf('https://ftp.ncbi.nlm....'` which appears to work before it doesn't.  That would be quite nice in the future!

Reply:
Local installs never run the reference data creation, because we make the output publicly available and they just download that, so thats not something we need to design for